src/Components/Facility/AddBedForm.tsx

```diff
@@ -124,10 +124,13 @@ export const AddBedForm = ({ facilityId, locationId, bedId }: Props) => {
       res?.ok && onSuccess("Bed updated successfully");
     } else {
       // Create
+
       const { res } = await request(routes.createFacilityBed, {
         body: { ...data, facility: facilityId, location: locationId },
       });
-      res?.ok && onSuccess("Bed(s) created successfully");
+      res?.ok && numberOfBeds > 1
+        ? onSuccess("Bed(s) created successfully")
+        : onSuccess("Bed created successfully");
```

Comment:
Let's keep representable texts in `en` locale files using `useTranslations`.

Also check-out the proper expected behaviour. Show "Beds" instead of "Bed(s)" for multiple bed creation.